docs/PaymentApi.md

```diff
@@ -426,6 +432,47 @@ try {
 }
 ```
 
+You can specify payment method by either `setPaymentMethod(method)` or `setPaymentMethodAsWallet()` at your payment flow.
```

Comment:
@yutafujiiPP san
This description would be out dated now as we remove these methods and create setter for paymentMethodType and paymentMethodId. can you review and change it?

Reply:
Fixed